styleguide/source/_patterns/01-molecules/06-components/20-testimonial/_testimonial.scss

```diff
@@ -0,0 +1,64 @@
+.testimonial {
+  position: relative;
+  width: 100%;
+  max-width: 1200px;
+  margin-bottom: 24px;
+
+  @include breakpoint($bp-med) {
+    display: flex;
+    flex-direction: row-reverse;
+    align-items: stretch;
+
+    &.testimonial-left {
+      flex-direction: row;
+    }
+  }
+}
+
+.testimonial_image {
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  min-height: calc(((100vw - 40px) / 3) * 2);
+
+  @include breakpoint($bp-med) {
+    min-height: calc(((100vw -  60px) / 2) / 3 * 2);
+    min-width: 50%;
+    flex-basis: 50%;
+  }
+
+  @include breakpoint($bp-large) {
+    min-height: 400px;
+  }
+}
+
+.testimonial_text {
+  padding: 30px;
```

Comment:
At mobile sizes, I think you want to step the padding down a bit (to like 15px maybe) and set a min height of maybe 400px on the image. Just a little something to keep the text from overwhelming the entire image. I know they've imagine the layouts kinda oddly (going from landscape to portrait depending on the screen size), but I think we can maybe get it a little closer to what they envisioned.

Wireframe vs styleguide
![mobileviews](https://user-images.githubusercontent.com/28711067/27403510-4f930fa2-5690-11e7-86b3-67ac1d8b32c3.jpg)